pkg/virt-operator/kubevirt_test.go

```diff
@@ -2624,6 +2635,181 @@ var _ = Describe("KubeVirt Operator", func() {
 			Expect(kvTestData.resourceChanges["daemonsets"][Patched]).To(Equal(0))           // namespace unpatched
 		})
 
+		Context("virt-api replica count", func() {
+			var kvTestData KubeVirtTestData
+			const (
+				CustomizedReplicas              int32 = 4
+				expectedReplicasForLargeCluster int32 = 100
+			)
+
+			BeforeEach(func() {
+				kvTestData = KubeVirtTestData{}
+				kvTestData.BeforeTest()
+				DeferCleanup(kvTestData.AfterTest)
+
+				var createdNodes []k8sv1.Node
+
+				for i := 0; i < 1000; i++ {
+					node := k8sv1.Node{
+						ObjectMeta: metav1.ObjectMeta{
+							Name: fmt.Sprintf("testnode-%d", i),
+						},
+					}
+					createdNodes = append(createdNodes, node)
+
+					_, err := kvTestData.kubeClient.CoreV1().Nodes().Create(context.TODO(), &node, metav1.CreateOptions{})
+					Expect(err).ToNot(HaveOccurred())
+				}
+
+				kvTestData.kubeClient.Fake.PrependReactor("list", "nodes", func(action testing.Action) (handled bool, obj runtime.Object, err error) {
+					return true, &k8sv1.NodeList{Items: createdNodes}, nil
+				})
+
+				nodes, err := kvTestData.virtClient.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				Expect(nodes.Items).To(HaveLen(1000))
```

Comment:
I think this is redundant, here you are testing the fake client itself, it seems to be out of scope.